Jenkinsfile

```diff
@@ -0,0 +1,65 @@
+pipeline {
+  agent { label 'executor-v2' }
+
+  options {
+    timestamps()
+    buildDiscarder(logRotator(numToKeepStr: '30'))
+    skipDefaultCheckout()  // see 'Checkout SCM' below, once perms are fixed this is no longer needed
+  } 
+
+  stages {
+    stage('Checkout SCM') {
+      steps {
+        sh 'sudo chown -R jenkins:jenkins .'  // bad docker mount creates unreadable files TODO fix this
```

Comment:
Bummer, here too? We really need to figure out these Docker volume mount permissions issues soon-ish. This is affecting multiple projects and making our pipelines more verbose.